admin/src/components/Users/ChangePassword.test.js

```diff
@@ -40,7 +40,7 @@ describe('(Component) Change Password', () => {
 
     const form = component.find('form').at(0);
     form.simulate('submit');
-    expect(onSubmit).toHaveBeenCalled();
+    // expect(onSubmit).toHaveBeenCalled();
```

Comment:
Same here, if we comment this out, then we don't need the test at all. And maybe we don't since it looks like testing `informed` behavior. Before we were testing our `<form>` to cause `onSubmit` to be called. now I don't think we need this.